tsconfig.settings.json

```diff
@@ -3,14 +3,13 @@
     "target": "es2022",
     "module": "node16",
     "moduleResolution": "node16",
+    "lib": ["es2022"],
```

Comment:
Isn't this already implied with `target: es2022`?

Reply:
No. `"target": "es2022"` implies `"lib": ["es2022", "dom", "dom.iterable"]`. 🤷‍♀️

Reply:
Of course 🤷